geom.go

```diff
@@ -415,6 +416,31 @@ func NewPolygonFromFlatPoints (shell []float64) (*Geometry, error){
 	return PolygonFromGeom(ext)
 }
 
+func (geom *Geometry) PolygonToFlatPoints(out []float64) ([]float64, error) {
+	shell, shellErr := geom.Shell()
+	if shellErr != nil {
+		return nil, shellErr
+	}
+	ptr := cGEOSGeom_getCoordSeq(shell.g)
+	if ptr == nil {
+		return nil, Error()
+	}
+	//cs := coordSeqFromPtr(ptr)
+	cs := &coordSeq{c: ptr}
+	nCoordinates, sizeErr := cs.size()
+	runtime.KeepAlive(cs)
+	if sizeErr != nil {
+		return nil, sizeErr
+	}
+	flatSize := nCoordinates * 2
```

Comment:
Why not using here the same trick you used before to multiply by 2? `>> 1`?

Reply:
I suppose you mean << 1, fixed